hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncClientScanner.java

```diff
@@ -188,6 +188,7 @@ private void startScan(OpenScannerResponse resp) {
   private CompletableFuture<OpenScannerResponse> openScanner(int replicaId) {
     return conn.callerFactory.<OpenScannerResponse> single().table(tableName)
       .row(scan.getStartRow()).replicaId(replicaId).locateType(getLocateType(scan))
+      .priority(scan.getPriority())
```

Comment:
This absence was a bug, no?

Reply:
Yes, this was a bug. I caught it in my investigation into these tests. Should I be filing a separate JIRA for that, or is it ok to include here?